packages/suite/src/hooks/wallet/useRbfForm.ts

```diff
@@ -199,7 +210,7 @@ export const useRbf = (props: Props) => {
         if (ctxState.account?.networkType !== 'bitcoin' || !composedLevels) return;
         const { selectedFee, setMaxOutputId, outputs } = getValues();
         const tx = composedLevels[selectedFee || 'normal'];
-        if (tx.type === 'error' && tx.error === 'NOT-ENOUGH-FUNDS') {
+        if (tx?.type === 'error' && tx.error === 'NOT-ENOUGH-FUNDS') {
```

Comment:
Some comment explaining the need for optional chaining might be handy in the future.

Reply:
https://github.com/trezor/trezor-suite/pull/7378/commits/1a974cb4ef4acb227405854e57337244c6719afe